docs/developer-docs/latest/setup-deployment-guides/deployment.md

```diff
@@ -156,12 +156,33 @@ We highly recommend using [pm2](https://github.com/Unitech/pm2/) to manage your
 
 If you need a server.js file to be able to run `node server.js` instead of `npm run start` then create a `./server.js` file as follows:
 
+<code-group>
+
+<code-block title="JAVASCRIPT">
+
 ```js
-const strapi = require('@strapi/strapi');
+// path: `./server.js`
 
+const strapi = require('@strapi/strapi');
 strapi(/* {...} */).start();
 ```
 
+</code-block>
+
+<code-block title="TYPESCRIPT">
+
+```ts
+// path: `./server.js`
+
+const strapi = require('@strapi/strapi');
+const app = strapi({ distDir: '<path_to_your_out_dir>' });
```

Comment:
Just a suggestion to keep consistency with the JS example
```suggestion
const app = strapi({ distDir: '<path_to_your_out_dir>' }).start();
```

Reply:
removed code example